src/components/ModelDialog/ModalDialog.tsx

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (C) 2022 -2023 Vereniging van Nederlandse Gemeenten, Gemeente Amsterdam
+import type { ReactNode } from 'react'
+
+import { ModalContent } from './components/ModalContent'
+import { ModalHeader } from './components/ModalHeader'
+import { StyledModal } from './styled'
+
+export type ModalDialogProps = {
+  onClose: () => void
+  title: string
+  onConfirm?: () => void
+  children?: ReactNode
+  isConfirmation?: boolean
+  description?: string
+}
+
+const ModalDialog = ({
+  onClose,
+  onConfirm,
+  title,
+  isConfirmation,
+  description,
+  children,
+}: ModalDialogProps) => (
+  <StyledModal open onClose={onClose}>
+    <ModalHeader title={title} onClose={onClose} />
```

Comment:
Why is this modal header necessary? The `StyledModal` also has a header looks pretty much the same...

Reply:
It was already in there. I didn't add it, so I don't know why it was originally added. I've kept it, so I can style it separately from for instance Filter (which also uses Modal).